code/modules/clothing/suits/aliens/seromi_cit.dm

```diff
@@ -0,0 +1,275 @@
+// Standard Cloaks
+/obj/item/clothing/suit/storage/teshari/cloak_cit
```

Comment:
all the _cit references should probably be renamed to _vr, in the filenames too.

Reply:
Was just noting where they came from but no problem, will simply leave a comment